lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -221,6 +222,11 @@ IncrementalJIT::getSymbolAddressWithoutMangling(llvm::StringRef Name,
     if (RuntimeDyld::SymbolInfo SymInfo = m_ExeMM->findSymbol(Name))
       return llvm::orc::JITSymbol(SymInfo.getAddress(),
                                   llvm::JITSymbolFlags::Exported);
+#ifdef LLVM_ON_WIN32
+    if (const void* Sym = platform::DLSym(Name))
```

Comment:
According to [this post](http://www.drdobbs.com/dynamic-plug-ins-for-c/184401819), on Windows, exported symbols are like Global symbols on UNIX. And according to the [LoadLibrary function documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/ms684175(v=vs.85).aspx): 
> LoadLibrary can be used to load a library module into the address space of the process and return a handle that can be used in GetProcAddress to get the address of a DLL function. 
But I'm not sure how this could answer the original question...